### PR TITLE
Plug the new Windows installer, and minor rework of the page

### PR DESCRIPTION
**Pull request by tjvantoll**

### CODE REVIEW

File: start/quick-setup.md [hunk 1]
```diff
@@ -10,12 +10,8 @@ previous_url: /setup/quick-setup
 
 With the open-source NativeScript command-line interface and an IDE or text editor of your choice, you can create, develop, store and build your apps entirely locally, free of charge and anonymously. Let’s look at how to set up the CLI for development.
 
-
-> **NOTE**: The steps on this page are quick setup steps intended for users new to mobile development. If you have existing mobile experience, if you’re on Linux, or if you want full control of the installation process, refer to one of the advanced setup guides below, which walk you through manually setting up your environment for NativeScript development.
-> * [Advanced setup: Windows](/start/ns-setup-win)
-> * [Advanced setup: macOS](/start/ns-setup-os-x)
-> * [Advanced setup: Linux](/start/ns-setup-linux)
-
+> **NOTE**: If you’re on Windows you might be interested in trying our new **Windows installer** instead of going through the steps on this page. The installer is in beta, so [let us know if you hit any issues](https://github.com/nativescript/nativescript-cli/issues).
```
Comment (tjvantoll) [inline start/quick-setup.md:13]: Should we ask for people to report issues? And if so what’s the correct venue for that?

Comment (rdlauer) [reply start/quick-setup.md:13]: Maybe we should set up a dedicated place in the forum?

Comment (tjvantoll) [reply start/quick-setup.md:13]: I really like the idea of a dedicated forum location. It keeps some noise out of GitHub issues and invites the community in to assist. I want to make sure engineering is ok with this approach before I push it out though.

**Comment by tjvantoll** [comment]
Closed with d64c8196233c86886de20c3dbc8740f652ca0299.

**Pull request closed without merging**